src/pretix/control/views/event.py

```diff
@@ -732,9 +733,12 @@ def post(self, request, *args, **kwargs):
                 idx = matched.group('idx')
                 if idx in self.supported_locale:
                     with language(self.supported_locale[idx], self.request.event.settings.region):
-                        msgs[self.supported_locale[idx]] = markdown_compile_email(
-                            v.format_map(self.placeholders(preview_item))
-                        )
+                        if 'subject' in k:
+                            msgs[self.supported_locale[idx]] = bleach.clean(v).format_map(self.placeholders(preview_item))
+                        else:
+                            msgs[self.supported_locale[idx]] = markdown_compile_email(
+                                v.format_map(self.placeholders(preview_item))
+                            )
```

Comment:
Would it make sense to `bleach.clean` after paceholder-interpolation? And also make that `if` checking for mail_subject more specific? E.g.
```suggestion
                        v = v.format_map(self.placeholders(preview_item))
                        if k.startswith('mail_subject_'):
                            msgs[self.supported_locale[idx]] = bleach.clean(v)
                        else:
                            msgs[self.supported_locale[idx]] = markdown_compile_email(v)
```

Reply:
> Would it make sense to bleach.clean after paceholder-interpolation?

Nope! The placeholders, in this case, will contain HTML, namely the snippet that makes them yellow and gives them an "I am a placeholder" tooltip.

> And also make that if checking for mail_subject more specific?

Yeah, maybe.